projects/chomsky/src/utils/formats.ts

```diff
@@ -76,6 +76,22 @@ export class Formats {
     return new Intl.DateTimeFormat([this.locale, 'en-US'], options).format(_value);
   }
 
+  public formatTime(value: any, format?: string | Intl.DateTimeFormatOptions): string {
+    const _value = value === null || value === undefined || value === '' ? new Date() : new Date(value);
+    const shortHands = mergeDeep({}, this.defaults.date);
```

Comment:
I think 81 - 88 are complex enough to be their own function. Would it make sense to break that out?

Reply:
externalized into own function